hnn_core/network.py

```diff
@@ -371,25 +383,26 @@ def _parnet_connect(self):
         from .parallel import pc
 
         # loop over target zipped gids and cells
-        # cells has NO extinputs anyway. also no extgausses
+        # cells has NO common feeds anyway. also no extgausses
+        # XXX no comprendo commento
```

Comment:
lol, this is why I just remove comments unless they really help comprehension :)